support-frontend/assets/pages/contributions-landing/contributionsLandingReducer.ts

```diff
@@ -43,7 +42,6 @@ interface FormData extends UserFormData {
 	checkoutFormHasBeenSubmitted: boolean;
 }
 export interface StripePaymentRequestButtonData {
-	paymentMethod: 'none' | StripePaymentMethod;
```

Comment:
presumably tsc would complain if this was still being used anywhere else, so should be fine!